### PR TITLE
add e2e test for kube-storage-migrator

### PR DESCRIPTION
**Pull request by caesarxuchao**
/assign @krzyzacy

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml [hunk 1]
```diff
@@ -9,3 +9,20 @@ presubmits:
...
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181019-3f9819d46-master
+        args:
+        - "--upload=gs://kubernetes-jenkins/logs"
+        - --
+        - --extract=ci/latest
+        - --test-cmd=../test/e2e/test-cmd.sh
```
Comment (krzyzacy) [inline config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: (you will still need other flags to bring up a cluster, just copy from the [gce job](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml#L125-L148)?)

Comment (caesarxuchao) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: Done.

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml [hunk 2]
```diff
@@ -9,3 +9,27 @@ presubmits:
         command:
         - make
         - test
+  - name: pull-kube-storage-version-migrator-e2e
+    decorate: true
+    always_run: true
+    # TODO: remove this when the test is stable
+    optional: true
+    labels:
+      preset-service-account: "true"
```
Comment (caesarxuchao) [inline config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: Where are these labels being used?

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml [hunk 3]
```diff
@@ -9,3 +9,27 @@ presubmits:
...
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181019-3f9819d46-master
+        args:
+         - --timeout=70
+         - --bare
```
Comment (krzyzacy) [inline config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: you still need to checkout your repo to get the script

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml (hunk 2 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: see https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L437-L455, you need them :-)

File: config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml (hunk 1 above)
Comment (krzyzacy) [reply config/jobs/kubernetes-sigs/kube-storage-version-migrator/presubmits.yaml]: add a timeout?

**Pull request merged**